packages/headless/src/api/analytics/analytics.ts

```diff
@@ -30,11 +38,19 @@ export class AnalyticsProvider implements SearchPageClientProvider {
       responseTime: this.responseTime,
       results: this.mapResultsToAnalyticsDocument(),
       numberOfResults: this.numberOfResults,
+      getBaseMetadata: this.getBaseMetadata(),
     };
   }
 
   public getBaseMetadata() {
```

Comment:
Using this method does also cause the context to be copied to a `getBaseMetadata` attribute inside the analytics request. I'm not sure if this causes any problems.